setup-stage1.sh

```diff
@@ -151,23 +151,17 @@ printf -v FFXIV_ENVIRON_FINAL '%s\nexport PATH=%q:$PATH' "$FFXIV_ENVIRON_FINAL"
 
 # Check for wine already being setcap'd, and fail if so.
 if [[ "$(getcap "$PROTON_PATH")" != "" ]]; then
-    error "Detected that you're running this against an already configured Proton (the binary at path \"$PROTON_PATH\" has capabilities set already)."
-    error "You must run this script against a fresh proton install, or else the LD_LIBRARY_PATH environment variable configured by your runtime cannot be detected."
-    exit 1
-fi
-
-if [[ "$(echo "$FFXIV_ENVIRON_FINAL" | grep 'export LD_LIBRARY_PATH=')" == "" ]]; then
-    warn "Unable to determine runtime LD_LIBRARY_PATH."
-    warn "This may indicate something strange with your setup."
-    warn "Continuing is not advised unless you know how to fix any issues that may come up related to missing libraries."
+    error "Detected that you're running this against an already configured wine (the binary at path \"$PROTON_PATH\" has capabilities set already)."
+    error "You must run this script against a fresh wine install, or else the LD_LIBRARY_PATH environment variable configured by your runtime cannot be detected."
     exit 1
 fi
```

Comment:
This entire block can probably be dropped. There should be no issue running against an already setcapped binary since the LD_LIBRARY_PATH isn't required since we're not patching the RPATH for XLCore.